poetry.lock

```diff
@@ -48,1165 +48,1207 @@ python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
+"aws-cdk.cx-api" = "1.134.0"
+"aws-cdk.region-info" = "1.134.0"
+constructs = ">=3.3.69,<4.0.0"
+jsii = ">=1.46.0,<2.0.0"
 publication = ">=0.0.3"
 
 [[package]]
 name = "aws-cdk.aws-lambda-event-sources"
-version = "1.92.0"
+version = "1.134.0"
 description = "Event sources for AWS Lambda"
 category = "main"
 optional = true
 python-versions = ">=3.6"
 
 [package.dependencies]
-"aws-cdk.aws-apigateway" = "1.92.0"
-"aws-cdk.aws-dynamodb" = "1.92.0"
-"aws-cdk.aws-events" = "1.92.0"
-"aws-cdk.aws-iam" = "1.92.0"
-"aws-cdk.aws-kinesis" = "1.92.0"
-"aws-cdk.aws-lambda" = "1.92.0"
-"aws-cdk.aws-s3" = "1.92.0"
-"aws-cdk.aws-s3-notifications" = "1.92.0"
-"aws-cdk.aws-sns" = "1.92.0"
-"aws-cdk.aws-sns-subscriptions" = "1.92.0"
-"aws-cdk.aws-sqs" = "1.92.0"
-"aws-cdk.core" = "1.92.0"
-constructs = ">=3.2.0,<4.0.0"
-jsii = ">=1.24.0,<2.0.0"
+"aws-cdk.aws-apigateway" = "1.134.0"
+"aws-cdk.aws-dynamodb" = "1.134.0"
+"aws-cdk.aws-ec2" = "1.134.0"
+"aws-cdk.aws-events" = "1.134.0"
+"aws-cdk.aws-iam" = "1.134.0"
+"aws-cdk.aws-kinesis" = "1.134.0"
+"aws-cdk.aws-lambda" = "1.134.0"
+"aws-cdk.aws-s3" = "1.134.0"
+"aws-cdk.aws-s3-notifications" = "1.134.0"
+"aws-cdk.aws-secretsmanager" = "1.134.0"
+"aws-cdk.aws-sns" = "1.134.0"
+"aws-cdk.aws-sns-subscriptions" = "1.134.0"
+"aws-cdk.aws-sqs" = "1.134.0"
+"aws-cdk.core" = "1.134.0"
+constructs = ">=3.3.69,<4.0.0"
+jsii = ">=1.46.0,<2.0.0"
 publication = ">=0.0.3"
 
 [[package]]
 name = "aws-cdk.aws-lambda-python"
-version = "1.92.0"
+version = "1.134.0"
 description = "The CDK Construct Library for AWS Lambda in Python"
 category = "main"
 optional = true
 python-versions = ">=3.6"
 
 [package.dependencies]
-"aws-cdk.aws-ec2" = "1.92.0"
-"aws-cdk.aws-lambda" = "1.92.0"
-"aws-cdk.core" = "1.92.0"
-constructs = ">=3.2.0,<4.0.0"
-jsii = ">=1.24.0,<2.0.0"
+"aws-cdk.aws-ec2" = "1.134.0"
+"aws-cdk.aws-lambda" = "1.134.0"
+"aws-cdk.core" = "1.134.0"
+constructs = ">=3.3.69,<4.0.0"
+jsii = ">=1.46.0,<2.0.0"
 publication = ">=0.0.3"
 
 [[package]]
 name = "aws-cdk.aws-logs"
-version = "1.92.0"
+version = "1.134.0"
 description = "The CDK Construct Library for AWS::Logs"
 category = "main"
 optional = true
 python-versions = ">=3.6"
 
 [package.dependencies]
-"aws-cdk.aws-cloudwatch" = "1.92.0"
-"aws-cdk.aws-iam" = "1.92.0"
-"aws-cdk.aws-kms" = "1.92.0"
-"aws-cdk.aws-s3-assets" = "1.92.0"
-"aws-cdk.core" = "1.92.0"
-constructs = ">=3.2.0,<4.0.0"
-jsii = ">=1.24.0,<2.0.0"
+"aws-cdk.aws-cloudwatch" = "1.134.0"
+"aws-cdk.aws-iam" = "1.134.0"
+"aws-cdk.aws-kms" = "1.134.0"
+"aws-cdk.aws-s3-assets" = "1.134.0"
+"aws-cdk.core" = "1.134.0"
+"aws-cdk.cx-api" = "1.134.0"
+constructs = ">=3.3.69,<4.0.0"
+jsii = ">=1.46.0,<2.0.0"
 publication = ">=0.0.3"
 
 [[package]]
 name = "aws-cdk.aws-route53"
-version = "1.92.0"
+version = "1.134.0"
 description = "The CDK Construct Library for AWS::Route53"
 category = "main"
 optional = true
 python-versions = ">=3.6"
 
 [package.dependencies]
-"aws-cdk.aws-ec2" = "1.92.0"
```

Comment:
This doesn't look like part of this change. You probably want to check out the "Adding or updating Python dependencies" section in the readme.